gyp/.github/workflows/release-please.yml

```diff
@@ -1,7 +1,7 @@
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
I am not a fan of `master --> main` but it is all the rage these daze...
1. We need to document how contributors change their local forks.
2. We need to document how contributors change the 29 open PRs.
3. https://github.com/nodejs/node-gyp/blob/master/.github/workflows/release-please.yml#L6 needs to change.

Reply:
It's in https://github.com/nodejs/gyp-next. The change to main branch has been made in https://github.com/nodejs/gyp-next/pull/103 by @targos . It's out of scope for this repo :)

Reply:
> It's out of scope for this repo :)

👍

If that topic does come up here, or elsewhere in the Node ecosystem, most of the documentation around that from GitHub is collected here: https://github.com/github/renaming/

Reply:
Every contributor needs to [do this](https://docs.github.com/en/github/administering-a-repository/managing-branches-in-your-repository/renaming-a-branch#updating-a-local-clone-after-a-branch-name-changes) for 1.  How do they do 2.?

Reply:
![image](https://user-images.githubusercontent.com/2352663/118940418-fbebf800-b950-11eb-8e77-829d68e91e29.png)

Reply:
The destination branches of ___all 29 pull requests___ are changed automatically?!?

Like `commit into nodejs:master` at the top of this page:
```
gengjiawen wants to merge 1 commit into nodejs:master from gengjiawen:feat/gyp_bump
```

Reply:
I think it is easier to understand the [instructions](https://docs.github.com/en/github/administering-a-repository/managing-branches-in-your-repository/renaming-a-branch#updating-a-local-clone-after-a-branch-name-changes) for 1. with long flags rather than short flags... (And I added links to the git subcommand documentation)


<code>$ <a href="https://git-scm.com/docs/git-branch#Documentation/git-branch.txt--m">git branch --move</a> OLD-BRANCH-NAME NEW-BRANCH-NAME</code>
<code>$ <a href="https://git-scm.com/docs/git-fetch">git fetch</a> origin</code>
<code>$ <a href="">git branch --set-upstream-to</a>=origin/NEW-BRANCH-NAME NEW-BRANCH-NAME</code>
<code>$ <a href="https://git-scm.com/docs/git-remote#Documentation/git-remote.txt-emset-headem">git remote set-head</a> origin --auto</code>